server/controllers/profile.js

```diff
@@ -1,26 +1,171 @@
+const asyncHandler = require("express-async-handler");
 const fs = require("fs");
 const User = require("../models/User");
-const asyncHandler = require("express-async-handler");
+const Profile = require("../models/Profile");
 const cloudinary = require("../utils/cloudinaryHelper");
 
+// @route GET /profiles
+// @desc get all profiles
+// @access Public
+exports.loadProfiles = asyncHandler(async (req, res, next) => {
+  const profiles = await Profile.find({});
+
+  if (!profiles.length) {
+    res.status(404);
+    throw new Error("No Profiles found");
+  }
+  res.status(200).send(profiles);
+});
+
+//@route POST /profiles
+//@desc CREATE a new profile
+//@access Private
+exports.createProfile = asyncHandler(async (req, res, next) => {
+  const user = await User.findById(req.user.id);
+  const userId = user?._id;
+  if (!user) {
+    res.status(401);
+    throw new Error("Not authorized");
```

Comment:
Is this because it's not authorized or you can't find the user?

Reply:
Hey @mayanksuchde, when its possible can you answer the above question? :)

Reply:
Yes, it's suppose to be  not authorized.@bonneli Sorry for the late reply...